src/server.rs

```diff
@@ -366,14 +370,15 @@ fn send_messages(
             trace!("no changes to send for {:?}", client.id());
         }
 
-        if !mutate_message.is_empty() {
+        if !mutate_message.is_empty() || track_mutate_messages {
```

Comment:
Why do you want to send empty messages? Just to confirm they are empty?

Reply:
Yes, necessary to differentiate between missed ticks and ticks without changes.